openapi.yaml

```diff
@@ -1,279 +1,684 @@
+            A residence hall might restrict bathroom use or building access to
+            residents only.
+          type: boolean
+        giRestroomLocations:
+          description: >
+            Room numbers of gender inclusive restrooms within location.
+            Some restrooms may not be ADA compliant.
+          type: string
+          example: 0111A, 0101, 0102
+        synonyms:
+          type: array
+          items:
+            type: string
+            example: "Gymnasium"
+        openHours:
+          $ref: '#/components/schemas/DayHours'
+        distance:
+          description: 'Distance from given location in given distance unit.
+                This is only returned when the query parameters lat and lon are specified.
+                The default distance unit is mile '
+          type: number
+          format: float
+          example: 1.8
+        tags:
+          description: 'Tags associated with this location'
+          type: array
+          items:
             type: string
-            format: uri
-            description: A link to further information about the error
-            example: https://developer.oregonstate.edu/documentation/error-reference#1234
-  ErrorResult:
-    properties:
-      errors:
-        type: array
-        items:
-          $ref: '#/definitions/ErrorObject'
+            example: cultural-center
+        parkingZoneGroup:
+          description: 'Zonegroup of location if location is parking'
+          type: string
+          example: A1
+        evParkingSpaceCount:
+          description: 'Count of electric vehicle parking spaces if the location is parking'
+          type: integer
+        adaParkingSpaceCount:
+          description: 'Count of ADA parking spaces if the location is parking'
+          type: integer
+        motorcycleParkingSpaceCount:
+          description: 'Count of motorcycle parking spaces if the location is parking.'
+          type: integer
+        weeklyMenu:
+          description: 'Link to a weekly menu if the location is dining.'
+          type: string
+          format: url
+          example: https://app.uhds.oregonstate.edu/api/dining/weeklymenu/index?loc=1
+    ServicesAttributes:
+      properties:
+        name:
+          type: string
+          description: Name of the Service.
+        tags:
+          description: 'Tags associated with this location'
+          type: array
+          items:
+            type: string
+            example: recreation
+        openHours:
+          type: array
+          items:
+            $ref: '#/components/schemas/DayHours'
+    RelationshipsInLocations:
+      properties:
+        services:
+            $ref: '#/components/schemas/ServicesRelationship'
+    RelationshipsInServices:
+      properties:
+        locations:
+            $ref: '#/components/schemas/ServicesRelationship'
+    ServicesRelationship:
+      properties:
+        data:
```

Comment:
I'd consider adding a links object here to display self links for the related resource